ghcide/src/Development/IDE/GHC/Util.hs

```diff
@@ -288,9 +289,13 @@ instance Outputable SDoc where
 
 -- | Print a GHC value in `defaultUserStyle` without unique symbols.
 --
--- This is the most common print utility, will print with a user-friendly style like: `a_a4ME` as `a`.
+-- This is the most common print utility, and it will print with a user-friendly style like: `a_a4ME` as `a`.
```

Comment:
Maybe also include the fact that it unescapes characters? That's quite an important difference from other outputable functions. In fact... maybe we should use hlint to ban use of other outputable functions apart from this one?

Reply:
> Maybe also include the fact that it unescapes characters?

Done.

> maybe we should use hlint to ban use of other outputable functions apart from this one?

TBH I don't know if this is configurable. I think the scheme is to use `printOutputable` by default, and also allow other print functions to be used when needed.

Reply:
I mean, we can use `hlint` to ban functions, so we could do it. I don't know if it's a good idea, though.